cmd/skipper/main.go

```diff
@@ -103,6 +109,9 @@ func init() {
 	flag.StringVar(&metricsPrefix, "metrics-prefix", "", metricsPrefixUsage)
 	flag.BoolVar(&debugGcMetrics, "debug-gc-metrics", false, debugGcMetricsUsage)
 	flag.BoolVar(&runtimeMetrics, "runtime-metrics", true, runtimeMetricsUsage)
+	flag.StringVar(&applicationLog, "application-log", "", applicationLogUsage)
+	flag.StringVar(&applicationLogPrefix, "application-log-prefix", "", applicationLogPrefixUsage)
```

Comment:
I thought we wanted some other default here. `APP`? `APPLICATION`? `APPLICATION_LOG`?